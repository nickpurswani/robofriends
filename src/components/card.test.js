import React from 'react';
import { render } from '@testing-library/react';
import Card from './Card';

test('MyComponent snapshot', () => {
  const { asFragment } = render(<Card />);
  expect(asFragment()).toMatchSnapshot();
});
