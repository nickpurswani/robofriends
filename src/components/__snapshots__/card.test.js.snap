// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent snapshot 1`] = `
<DocumentFragment>
  <div
    class=" tc bg-light-green dib br4 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/undefined?200*200"
    />
    <div>
      <h2 />
      <p />
    </div>
  </div>
</DocumentFragment>
`;
